fluent/client/ws_client.go

```diff
@@ -114,12 +112,12 @@ func (wcf *DefaultWSConnectionFactory) New() (ext.Conn, error) {
 	}
 
 	conn, resp, err := dialer.Dial(wcf.URL, header)
+
 	if resp != nil && resp.Body != nil {
 		bodyBytes, readErr := io.ReadAll(resp.Body)
```

Comment:
I think we should first check for `readErr` here as well. This approach is consistent throughout the code base, so we should apply that principle here. It aligns with Go language best practices too.

```
bodyBytes, readErr := io.ReadAll(resp.Body)
if readErr != nil {
    return nil, readErr
}
```